exporters/zipkin/zipkin_test.go

```diff
@@ -375,3 +375,17 @@ func TestLogrFormatting(t *testing.T) {
 	got := buf.String()
 	assert.Equal(t, want, got)
 }
+
+func TestWithHeaders(t *testing.T) {
```

Comment:
Could you add a test checking that the specified headers are indeed added to the outgoing request?